lib/redux/reducers/saved_search/saved_search_list_reducer.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:pass_emploi_app/redux/actions/saved_search_actions.dart';
+
+import '../../states/state.dart';
+
+class SavedSearchListReducer {
```

Comment:
Le `Reducer<RESULT>` ne fait-il pas le job ? Vous avez vraiment besoin de ce Reducer specifique là ?

Reply:
J'avoue ne pas trop comprendre l'intérêt du Reducer générique. J'ai l'impression qu'il complexifie le code par les côté génériques et pour les typedef qu'il fait utiliser. C'est assez difficile à débuger également. Et au final le gain de temps est assez faible puisque le reducer est simple à écrire. 
Mais je suis chaud pour en discuter avec le reste de l'équipe.

Reply:
OK on en reparle plus tard alors ;) Forget it here :)